autoload/neoformat/run.vim

```diff
@@ -43,7 +43,7 @@ function! s:on_stdout(job_id, data) abort
     call extend(job.stdout, a:data)
 endfunction
 
-function! s:on_stdout_vim(job, data) abort
+function! s:on_stdout_vim(job, data, event) abort
```

Comment:
Vim's job control implementation doesn't use a third argument `event`, and will through some errors if this is present.